drivers/soundwire/generic_bandwidth_allocation.c

```diff
@@ -284,7 +333,7 @@ static int sdw_compute_port_params(struct sdw_bus *bus)
 
 	/* Compute transport parameters for grouped streams */
 	ret = sdw_compute_group_params(bus, params,
-				       &group.rates[0], group.count);
+				       &group.rates[0], &group.lanes[0], group.count);
```

Comment:
wondering if the order of patches should be modified to add first global changes to the allocation to make sure there's no regression by introducing all those modified loops in single-lane configuration, and then add the part where the multiple lanes are managed by RT722.

Reply:
> wondering if the order of patches should be modified to add first global changes to the allocation to make sure there's no regression by introducing all those modified loops in single-lane configuration, and then add the part where the multiple lanes are managed by RT722.

Yes, done

Reply:
> > wondering if the order of patches should be modified to add first global changes to the allocation to make sure there's no regression by introducing all those modified loops in single-lane configuration, and then add the part where the multiple lanes are managed by RT722.
> 
> Yes, done

Actually, no. `lane_maps[SDW_MAX_LANES];` is added in the `soundwire: mipi_disco: read lane mapping properties from ACPI` commit, and will be used in the follow-up commits. We can't move `soundwire: mipi_disco: read lane mapping properties from ACPI` to the back.